### PR TITLE
Czech translation for v1.3

### PR DESCRIPTION
**Pull request by dvorapa**
Re-targetted #10935 to master

### CODE REVIEW

**Comment by rroshan1** [comment]
Thanks a lot @dvorapa for your efforts.
@kvarel and @martinstarman - Please let us know if you have any feedback on these Czech translations.

File: src/nls/cs/strings.js [hunk 1]
```diff
@@ -88,6 +88,7 @@ define({
     // Řetězce chyb nastavení aplikace
     "ERROR_PREFS_CORRUPT_TITLE"         : "Chyba při čtení nastavení",
     "ERROR_PREFS_CORRUPT"               : "Soubor s nastavením není platný soubor typu JSON. Soubor bude otevřen, abyste mohli opravit formát. Pro uložení změn budete muset restartovat aplikaci {APP_NAME}.",
+    "ERROR_PROJ_PREFS_CORRUPT"          : "Soubor s preferencemi vašeho projektu neodpovídá vylidnímu JSON formátu. Tento soubor bude otevřen, abyste mohl formát opravit. K provedení změn bude nutné znovu načíst projekt.",
```
Comment (martinstarman) [inline src/nls/cs/strings.js]: There is a typo in word `vylidnímu`. Right is `validnímu`.

**Comment by martinstarman** [comment]
There is one typo, the rest of changes looks good.

File: src/nls/cs/strings.js (hunk 1 above)
Comment (dvorapa) [inline src/nls/cs/strings.js]: Corrected, thank you

**Comment by rroshan1** [comment]
Thanks @martinstarman for the review.
Merging it.

**Pull request merged**